doc/source/overview/high_level.md

```diff
@@ -101,7 +101,7 @@ appropriate. However, human bias itself is still an element. Hence, if the model
 data, we are vulnerable to using explainability to justify relations in the data that may not be accurate. Consider:
 > _"Before launching the model, risk analysts are asked to review the Shapley value explanations to ensure that the
 > model exhibits expected behavior (i.e., the model uses the same features that a human would for the same task)."_
-> &mdash; <cite>[Explainable Machine Learning in Deployment](https://dl.acm.org/doi/pdf/10.1145/3351095.3375624)</cite>
+> &mdash; <cite>[Explainable Machine Learning in Deployment](https://arxiv.org/abs/1909.06342)</cite>
```

Comment:
Worth using this link instead? https://dl.acm.org/doi/abs/10.1145/3351095.3375624